### PR TITLE
Feat(updates-ssr): Updates packages

### PR DESCRIPTION
**Pull request by rapahaeru**
https://jirasoftware.catho.com.br/browse/CCOM-445

[ ] updates quantum
[ ] updates catho components
[ ] updates readme
[ ] updates render page
[ ]  implements bundle analyzer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,10 @@
   "private": true,
   "dependencies": {
     "@babel/core": "^7.4.5",
-    "@catho-private/catho-components": "^5.8.2",
+    "@catho-private/catho-components": "^5.10.0",
     "@catho/quantum": "^9.0.1",
```
Comment (douglasjr97) [inline package.json:10]:
```suggestion
    "@catho/quantum": "^9.2.0",
```
should update quantum version.

**Review by douglasjr97 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hawkjr97) [inline package.json]:
```suggestion
    "@catho/quantum": "^9.2.0",
```

should update quantum version.

**Review by hawkjr97 — CHANGES_REQUESTED** [review]
[x] updates quantum
[x] updates catho components
[] updates readme
[] updates render page
[]  implements bundle analyzer

**Comment by hawkjr97** [comment]
Why can't I move the quantum page?

image

File: README.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 | lib to simulate  | version | path builded in application                  |
 | ---------------- | ------- | -------------------------------------------- |
-| Quantum          | 9.0.1   | node_modules/@catho/Quantum                  |
-| Catho-components | 5.8.2  | node_modules/@catho-private/catho-components |
+| Quantum          | 9.2.0   | node_modules/@catho/Quantum                  |
```
Comment (hawkjr97) [inline README.md]:
```suggestion
| Quantum          | 9.4.0   | node_modules/@catho/Quantum
```

9.4.0  Wouldn't it be the current version?

**Review by hawkjr97 — CHANGES_REQUESTED** [review]
## Review
[x] updates quantum
[x] updates catho components
[x] updates readme
[ ] updates render page
[ ]  implements bundle analyzer

File: README.md (hunk 2 above)
Comment (MarcosViniciusPC) [reply README.md]: At the time the quantum version was updated this was the latest version. There was a merger in quantum and now it is outdated. In order to make this as standardized as possible with the performance tests, I think it's interesting to keep the quantum in the 9.1.1 version that was used in the test. I will adjust

**Comment by MarcosViniciusPC** [comment]
> Why can't I move the quantum page?
> 
> image

It's because of the Dialog component that was rendered on the screen

**Review by hawkjr97 — APPROVED** [review]
## Review
[x] updates quantum
[x] updates catho components
[x] updates readme
[x] updates render page
[x]  implements bundle analyzer

**Pull request merged**